topics/mahapps/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: mahapps-metro, metro
+created_by: Paul Jenkins
+display_name: "MahApps.Metro"
+github_url: https://github.com/MahApps/MahApps.Metro
+logo: mahapps.png
+released: January 29, 2011
+short_description: "A toolkit for creating metro-style WPF applications."
+topic: mahapps
+url: http://mahapps.com/
+---
+The goal of MahApps.Metro is to allow developers to quickly and easily cobble together a `Metro` or `Modern UI` for their own `WPF` applications, with minimal effort.
```

Comment:
suggest rephrasing to focus on what it is, e.g.

> MahApps.Metro is a framework that allows developers to cobble together a `Metro` or `Modern UI` for their own `WPF` applications with minimal effort.

Reply:
@nayafia done